### PR TITLE
Eri9600/map image layer tables

### PR DESCRIPTION
**Pull request by eri9600**
@TADraeseke Query Map Image Layer Tables sample as described here: https://devtopia.esri.com/runtime/common-samples/issues/562
Please review when you have time. Using arcgis 100.3.0.2080

### CODE REVIEW

File: java/map-image-layer-tables/src/main/java/com/esri/arcgisruntime/sample/mapimagelayertables/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    ArcGISMap map = new ArcGISMap(Basemap.createStreets());
+
+    // create a new ArcGISMapImageLayer with a Service Request Map Server and load tables and layers
+    ArcGISMapImageLayer serviceRequestMapImageLayer = new ArcGISMapImageLayer(getString(R.string.map_service));
+    serviceRequestMapImageLayer.loadTablesAndLayersAsync();
+
+    // initialize graphics overlay
+    GraphicsOverlay graphicsOverlay = new GraphicsOverlay();
+
+    //TODO: check if initial viewpoint is working
```
Comment (eri9600) [inline java/map-image-layer-tables/src/main/java/com/esri/arcgisruntime/sample/mapimagelayertables/MainActivity.java]: @TADraeseke can you check if the initial viewpoint is correct for you? for me it shows the whole map on the screen. I think it is only supposed to show North America and South America.

Comment (eri9600) [reply java/map-image-layer-tables/src/main/java/com/esri/arcgisruntime/sample/mapimagelayertables/MainActivity.java]: also it does not have a README, the issue link does not have one so I am not too sure what to add.

File: java/map-image-layer-tables/src/main/java/com/esri/arcgisruntime/sample/mapimagelayertables/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+
+    // inflate views from layout
+    mMapView = findViewById(R.id.mapView);
+    mCommentListView = findViewById(R.id.comment_list);
+
+    // initialize string list that will hold the comments for the mCommentListView
```
Comment (TADraeseke) [inline java/map-image-layer-tables/src/main/java/com/esri/arcgisruntime/sample/mapimagelayertables/MainActivity.java]: Don't need to refer to the exact variable name in comments, ie comment list view (without the m)

File: java/map-image-layer-tables/src/main/java/com/esri/arcgisruntime/sample/mapimagelayertables/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+            ArrayAdapter<String> adapter = new ArrayAdapter<>(MainActivity.this,
+                android.R.layout.simple_list_item_1,
+                commentList);
+            // add the adapter to the List View
+            mCommentListView.setAdapter(adapter);
+          } catch (InterruptedException | ExecutionException e) {
+            Log.e(TAG, "Result Failure");
+          }
+        });
+      }
```
Comment (TADraeseke) [inline java/map-image-layer-tables/src/main/java/com/esri/arcgisruntime/sample/mapimagelayertables/MainActivity.java]: else or else if to handle failed load

File: java/map-image-layer-tables/src/main/java/com/esri/arcgisruntime/sample/mapimagelayertables/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+                  Color.CYAN, 14);
+              // create a graphic using the service request point and marker symbol
+              Graphic requestGraphic = new Graphic(serviceRequestPoint, selectedRequestedSymbol);
+              // add graphic to the map and zoom the map view
+              graphicsOverlay.getGraphics().add(requestGraphic);
+              mMapView.setViewpointCenterAsync(serviceRequestPoint, 150000);
+            }
+          });
+        } catch (InterruptedException | ExecutionException e) {
+          Log.e(TAG,"Related Request Failure");
```
Comment (TADraeseke) [inline java/map-image-layer-tables/src/main/java/com/esri/arcgisruntime/sample/mapimagelayertables/MainActivity.java]: use the exception message in the error

File: java/map-image-layer-tables/src/main/res/layout/service_comments.xml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+              android:orientation="vertical"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:background="#FFFFFF">
+
+    <ListView android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:id="@+id/comment_list"
+              android:divider="#7F7F7F"
+              android:dividerHeight="1dp"/>
```
Comment (TADraeseke) [inline java/map-image-layer-tables/src/main/res/layout/service_comments.xml]: maybe add 'android:fadeScrollbars="false"' it wasnt' clear to me at first that the top window was a listview

File: java/map-image-layer-tables/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+                android:layout_height="match_parent"  tools:context=".MainActivity">
+
+    <!-- MapView -->
+    <com.esri.arcgisruntime.mapping.view.MapView
+            android:id="@+id/mapView"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
```
Comment (TADraeseke) [inline java/map-image-layer-tables/src/main/res/layout/activity_main.xml]: can end this block with a /> instead

**Comment by eri9600** [comment]
@TADraeseke I fixed the items you pointed out. Let me know if i missed anything else.

File: java/map-image-layer-tables/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -0,0 +1,4 @@
+<resources>
+    <string name="app_name">Map Image Layer Tables</string>
+    <string name="map_service"> "https://sampleserver6.arcgisonline.com/arcgis/rest/services/ServiceRequest/MapServer"</string>
```
Comment (TADraeseke) [inline java/map-image-layer-tables/src/main/res/values/strings.xml]: no quotes required around string

**Comment by TADraeseke** [comment]
Thanks @eri9600 just noticed unnecessary quotes around url in the strings.xml. After that, good to merge. Nice work!

**Comment by eri9600** [comment]
@TADraeseke thanks for the review, will merge now.

**Pull request merged**